luminaire/model/window_density.py

```diff
@@ -39,8 +39,8 @@ class WindowDensityHyperParams(BaseModelHyperParams):
         moving average method.
     
         .. Note :: ma_window_length should be small enough to maintain the stable structure of the training / scoring window
-            and large enough to remove the trend. The ideal size can be somewhere between (0.1 * window_length) and
-            (0.25 * window length).
+            and large enough to remove the trend. The ideal size can be somewhere between (0.01 * window_length) and
+            (0.25 * window length) depending on the data frequency.
```

Comment:
well well well, turns out you are actually using the multiline form (and correctly indented) :rofl: